EMS/admin-ui-bundle/src/Resources/views/bootstrap5/release/form.html.twig

```diff
@@ -32,14 +32,20 @@
 		{% if release is defined and release.status == 'wip' %}
 			<div class="box-footer">
 				<div class="btn-group">
-					{% include '@EMSAdminUI/bootstrap5/elements/get-button.html.twig' with {
-						'url':  path('emsco_release_add_revisions', {'release': release.id} ),
-						'label': 'release.actions.add_revisions'|trans,
+					{% include '@EMSCore/elements/get-button.html.twig' with {
+						'url':  path('emsco_release_add_revisions', {'release': release.id, 'type': 'publish' } ),
+						'label': 'release.actions.add_publish'|trans,
 						'icon': 'plus',
 						'btnType': 'primary',
 					}%}
+                    {% include '@EMSCore/elements/get-button.html.twig' with {
+                        'url':  path('emsco_release_add_revisions', {'release': release.id, 'type': 'unpublish' } ),
+                        'label': 'release.actions.add_unpublish'|trans,
+                        'icon': 'minus',
+                        'btnType': 'default',
+                    }%}
 					{% if release.revisions|length > 0 %}
-						{% include '@EMSAdminUI/bootstrap5/elements/post-button.html.twig' with {
```

Comment:
Idem

Reply:
Fixed in https://github.com/ems-project/elasticms/pull/766/commits/14199b747aa4887ac48157c154f1126b44a09e8c